frontend/ui/widget/inputdialog.lua

```diff
@@ -99,6 +102,21 @@ function InputDialog:init()
         }
     }
 
+    if self.description then
```

Comment:
Could you also document this feature in the docstring at the top so it can be included in koreader.rocks/doc. I feel like this is a very useful feature that many people might want to use in the future.